server/service/src/sync/mod.rs

```diff
@@ -83,3 +86,15 @@ impl ActiveStoresOnSite {
         self.stores.iter().map(|r| r.store_row.id.clone()).collect()
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SyncApiSettings {
+    pub server_url: String,
+    pub username: String,
+    pub password_sha256: String,
+    pub site_uuid: String,
+    pub app_version: String,
+    pub app_name: String,
+    pub sync_version: String,
```

Comment:
Thanks, yeah better place in API, ideally in some shared place like api/mod, api/v5/, api/v6, that's a bit too much to refactor right now, I think it would also mean we can move some of the errors and sync record shapes to a common place, add to [carry over](https://github.com/msupply-foundation/open-msupply/issues/2821)

`Relocate common API shapes to a shared space`